inference-engine/thirdparty/clDNN/src/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -811,9 +863,81 @@ void prepare_primitive_fusing::fuse_simple_primitives(program_impl &p) {
             if (parent2->is_type<convolution>() && !conv_supports_fusings(parent2->as<convolution>()))
                 return;
 
-            // This fusing can be extended to support peer node in any layout
-            bool merge_allowed = fused_node->get_users().size() == 1;
+            bool merge_allowed = true;
+            // If fused node is not convolution and fused node has multiple users,
+            //  follow the legacy checking rule
+            if (fused_node->is_type<convolution>() && fused_node->get_users().size() > 1) {
+                // Allowed new pattern: Eltw1, Act, Eltw2, Elt3, Elt4 are fused to Conv1
+                // * Conv1 -> Eltw1(Add) -> Act(Clamp) -> Eltw2(Mul) -> Eltw3(Div) -> Eltw4(Add) -> Conv2
```

Comment:
Currently eltwise is only fused if it is mul or sum right? Can it fuse div too?

Reply:
div means mul in eltwise name. I just copy yolov5 networks node name but I thinks it makes misunderstanding. I will change it.

Reply:
I upload update commit. thanks